### DISCUSSION TITLE
Hook to close a modal programmatically.

### DISCUSSION THREAD

**Discussion by codescrafter** [rainbow-me/rainbowkit#1276] [Ideas]
I am using a web3auth connector with the rainbow kit. When I select web3Auth from a list of available options, it opens its own modal, which we can close by clicking the cross icon. For a better user experience, I want to close rainbow kit modal as well when the close icon on the web3auth modal is clicked. For that purpose, I need hook that I can call to close the modal.
Thanks.

**Comment by DanielSinclair** [comment]
Tracking discussion for this feature request in #1347
